### PR TITLE
release: reduce codegen unit

### PR DESCRIPTION
**Pull request by BusyJay**
This is a temporary work around for rust-lang/rust#50199.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -118,3 +118,4 @@ codegen-units = 4
 lto = true
 opt-level = 3
 debug = true
+codegen-units = 2
```
Comment (siddontang) [inline Cargo.toml]: can it reduce the performance?

Comment (BusyJay) [reply Cargo.toml]: Maybe, but at least it won't OOM (on my machine and @Hoverbear's laptop) anymore.

**Comment by siddontang** [comment]
Maybe we need to do a simple benchmark for it. Like `make bench`.

@Hoverbear Does the Rust team has any plan to improve it?

File: Cargo.toml (hunk 1 above)
Comment (Hoverbear) [reply Cargo.toml]: It increases compile time, but I do not think performance of the end binary.

**Comment by Hoverbear** [comment]
@siddontang At the moment the issue https://github.com/rust-lang/rust/issues/50199 has a number of open questions which we need to respond to before we can even confirm it is a bug.

File: Cargo.toml (hunk 1 above)
Comment (overvenus) [reply Cargo.toml]: Could you add a TODO since it's a temporary work around?

**Comment by BusyJay** [comment]
ping

**Comment by siddontang** [comment]
LGTM

PTAL @Hoverbear @overvenus

**Comment by BusyJay** [comment]
Doesn't seem to be related to the changes, /run-all-tests

**Pull request merged**